src/view/utils.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import ViewPosition from './position';
+import ViewTreeWalker from './treewalker';
+
+/**
+ * Contains utility functions for working on view.
+ *
+ * @module engine/view/utils
+ */
+
+/**
+ * For given {@link module:engine/view/text~Text view text node}, it finds previous or next sibling that is contained
+ * in the same container element. If there is no such sibling, `null` is returned.
+ *
+ * @param {module:engine/view/text~Text} node Reference node.
+ * @param {Boolean} getNext If `true` next touching sibling will be returned. If `false` previous touching sibling will be returned.
```

Comment:
It'd be good to find it a better name... but I can't :D. `lookRight`?

Also, this is an optional param. Should be marked with `[]`. Maybe this will clarify its meaning (typically, boolean flags are optional).